apps/ejabberd/src/gen_mod.erl

```diff
@@ -63,6 +64,9 @@
 %%     undefined.
 -callback start(Host :: ejabberd:server(), Opts :: list()) -> any().
 -callback stop(Host :: ejabberd:server()) -> any().
+% -callback deps(Host :: ejabberd:server(), Opts :: proplists:list()) ->
```

Comment:
It would worth to add a word of explanation why the `deps` callback is commented.